app/gui/view/component-browser/component-group/src/lib.rs

```diff
@@ -55,6 +56,7 @@ use ensogl_shadow as shadow;
 
 pub mod entry;
 pub mod icon;
+pub mod multi;
```

Comment:
I'd name it 'set'. The word 'multi' is not a noun and doesnt express the intention as good as "set" imo. Do you agree?